### DISCUSSION TITLE
Supervising during mentoring session

### DISCUSSION THREAD

**Discussion by Saupia** [teamalgoritma/community#456] [Q&A] [answered]
Is it possible to have a mentor supervise any LBB project during a mentoring session on Friday (in any weekly session)?

**Comment by VictorNugraha** [comment, marked as answer]
Dear @Saupia,

Yes it is possible to ask our mentor about your LBB project, but please keep this in mind, mentoring session on Friday is devoted to discussing that week's material, so our mentor will prioritize to answer the questions that related to those week's materials. Thank you.

Best Regards,
Team Algoritma

**Reply by Saupia** [reply]
Roger that. Thank you.
